server/routerlicious/packages/services-client/src/test/historian.spec.ts

```diff
@@ -130,10 +129,10 @@ describe("Historian", () => {
 		credentials: ICredentials,
 		additionalQueryParams?: any,
 	) => {
-		return `${endpoint}${path}?${qs.encode({
+		return `${endpoint}${path}?${new URLSearchParams({
```

Comment:
Just wondering if it's behavior-compatible with qs.encode() *and* qs.stringify(). Also, how far back its availability in browsers goes. We got bit recently by structuredClone not being available in some browsers that Loop supports, and since drivers depend on this package and end up running in browser contexts, we should double check.

Reply:
Encode and stringify are equal in the polyfill (https://www.npmjs.com/package/querystring?activeTab=code) and are specified as equivalent in the nodejs documentation (https://nodejs.org/api/querystring.html#querystringencode)

URLSearchParams is used pretty extensively already throughout the codebase on the client side, so if there were an issue I imagine we'd have hit it already.  structuredClone is a good bit newer than URLSearchParams.

https://caniuse.com/?search=urlsearchparams
https://caniuse.com/?search=structuredclone